ChakraCore Samples/CrossPlatform/README.md

```diff
@@ -0,0 +1,28 @@
+### Embedding ChakraCore on Linux and macOS
```

Comment:
To be consistent with existing readmes here, I would put something like [this](https://github.com/Microsoft/Chakra-Samples/blob/master/ChakraCore%20Samples/Hello%20World/C%2B%2B/README.md) or below, 

# ChakraCore Hello-world Sample
This sample shows how to embed ChakraCore with [JavaScript Runtime (JSRT) APIs](http://aka.ms/corejsrtref) on Linux/OS X, and complete the following tasks:

1. Initializing the JavaScript runtime and creating an execution context.
2. Running scripts and handling values.
3. Printing out script results.

## Requirements
This sample requires Ubuntu 16.04 LTS or [insert OS X system requirement]. 

## Build and run the sample
1. [Clone](https://github.com/Microsoft/ChakraCore) and [build](https://github.com/Microsoft/ChakraCore/wiki/Building-ChakraCore) ChakraCore on designated platform. Specify `--static` and `--debug` flag while running the command `./build.sh --static --debug`.  
2. Copy the files of this sample into `ChakraCore/CrossPlatform` folder.  
3. Cd to `ChakraCore/CrossPlatform` folder, and build with `make BUILD_TYPE=Debug`. 
4. Run with `./sample.o`! 
 
## Help us improve our samples
Help us improve out samples by sending us a pull-request or opening a [GitHub Issue](https://github.com/Microsoft/Chakra-Samples/issues/new).

Copyright (c) Microsoft. All rights reserved.  Licensed under the MIT license. See LICENSE file in the project root for full license information.